python/1.0.0/sample_manage_blocklist.py

```diff
@@ -0,0 +1,335 @@
+# coding: utf-8
+
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+def create_or_update_text_blocklist():
+    # [START create_or_update_text_blocklist]
```

Comment:
Please add comment like dotnet code to let doc team easily find corresponding code block.